Resources/Computing/PoolComputingElement.py

```diff
@@ -194,6 +194,38 @@ def getCEStatus(self, jobIDList=None):
     result['AvailableProcessors'] = self.processors - processorsInUse
     return result
 
+  def getDescription(self):
+    """ Get CE description as a dictionary
+    """
+    result = super(PoolComputingElement, self).getDescription()
+    if not result['OK']:
+      return result
+    ceDict = result['Value']
+
+    ceDictList = []
+    if self.ceParameters.get('MultiProcessorStrategy'):
```

Comment:
Where is this set?

Reply:
This is in the ComputingElement configuration. Either in PoolComputingElement in general or rather in he local CEDefaults as usually done by a pilot

Reply:
I just grepped the code (and the docs) and could not find it anywhere. Is it a matter of documentation?

Reply:
I will add it to the docs